x-pack/plugins/aiops/public/components/spike_analysis_table/spike_analysis_table_groups.tsx

```diff
@@ -253,64 +235,52 @@ export const SpikeAnalysisGroupsTable: FC<SpikeAnalysisTableProps> = ({
+          if (valuesBadges.length >= MAX_GROUP_BADGES) break;
           valuesBadges.push(
             <>
               <EuiBadge
                 key={`${fieldName}-id`}
                 data-test-subj="aiopsSpikeAnalysisTableColumnGroupBadge"
                 color="hollow"
               >
-                <span>{`${fieldName}: `}</span>
+                <span>
+                  {(duplicate ?? 0) <= 1 ? '* ' : ''}
+                  {`${fieldName}: `}
+                </span>
                 <span style={{ color: visColors[2] }}>{`${fieldValue}`}</span>
               </EuiBadge>
               <EuiSpacer size="xs" />
             </>
           );
         }
-        if (repeatedValues.length > 0 || hasExtraBadges) {
-          valuesBadges.push(
-            <>
-              <EuiBadge
-                key={`$more-id`}
-                data-test-subj="aiopsSpikeAnalysisGroupsTableColumnGroupBadge"
-                color="hollow"
-              >
-                {hasExtraBadges ? (
-                  <>
-                    <FormattedMessage
-                      id="xpack.aiops.explainLogRateSpikes.spikeAnalysisTableGroups.moreLabel"
-                      defaultMessage="+{count, plural, one {# more field/value pair} other {# more field/value pairs}}"
-                      values={{ count: group.length - MAX_GROUP_BADGES }}
-                    />
-                    <br />
-                  </>
-                ) : null}
-                {repeatedValues.length > 0 && valuesBadges.length ? (
-                  <FormattedMessage
-                    id="xpack.aiops.explainLogRateSpikes.spikeAnalysisTableGroups.moreRepeatedLabel"
-                    defaultMessage="+{count, plural, one {# more field/value pair} other {# more field/value pairs}} also appearing in other groups"
-                    values={{ count: repeatedValues.length }}
-                  />
-                ) : null}
-                {repeatedValues.length > 0 && !valuesBadges.length ? (
-                  <FormattedMessage
-                    id="xpack.aiops.explainLogRateSpikes.spikeAnalysisTableGroups.onlyMoreRepeatedLabel"
-                    defaultMessage="{count, plural, one {# field/value pair} other {# field/value pairs}} also appearing in other groups"
-                    values={{ count: repeatedValues.length }}
-                  />
-                ) : null}
-              </EuiBadge>
-              <EuiSpacer size="xs" />
-            </>
-          );
-        }
+
+        valuesBadges.push(
+          <>
+            <EuiText
+              key={`$more-id`}
+              data-test-subj="aiopsSpikeAnalysisGroupsTableColumnGroupBadge"
+              color="subdued"
+              size="xs"
+            >
+              <FormattedMessage
+                id="xpack.aiops.explainLogRateSpikes.spikeAnalysisTableGroups.groupInfo"
+                defaultMessage="Showing {valuesBadges} out of {count} group items. {unique} items unique to this group."
```

Comment:
This will need editing as I just found out

<img width="303" alt="image" src="https://user-images.githubusercontent.com/7405507/226414166-49a360e5-7792-4962-bf78-72f0464137a7.png">

Reply:
I wonder if this message is needed at all if the number of items in the group is 5 or less? In my tests, a majority of the rows have 5 or less, so it's adding extra visual noise which I'm not convinced is needed.

<img width="428" alt="image" src="https://user-images.githubusercontent.com/7405507/226416995-84fb866f-8441-479b-a95f-e0ba4267d49f.png">

We already display `Items which are unique to a group are marked by an asterisk (*).` underneath the grouping control and also have a tooltip on the `*` in the expanded row. In the above example, I'd suggest the extra info is only needed for the bottom row where there are 9 items.

The word `group` in the first sentence probably isn't needed either e.g. Showing 5 out of 9 items. 3 items are unique to this group.

Reply:
Updated the group info text in [0726764](https://github.com/elastic/kibana/pull/153091/commits/0726764bbdb74aa640d599298f4f3ea908c86889).

- The text will now be only shown if there are more than 5 items.
- Split the 2 sentences into 2 `<FormattedMessage>` components, the second sentence about unique items will now only be shown if there are more than 5 unique items. 

Examples:

<img width="699" alt="image" src="https://user-images.githubusercontent.com/230104/226857407-7a537ebd-6fb5-4a88-b609-7eaa7ed07507.png">

<img width="608" alt="image" src="https://user-images.githubusercontent.com/230104/226857832-026cc673-cc32-409c-a2f0-ec9a122a339a.png">

Reply:
These changes look good.